install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -571,6 +571,9 @@ data:
 {{- if hasKey .Values.k8s "requireIPv6PodCIDR" }}
   k8s-require-ipv6-pod-cidr: {{ .Values.k8s.requireIPv6PodCIDR | quote }}
 {{- end }}
+{{- if .Values.endpointStatus.enabled }}
+  endpoint-status: {{ required "endpointStatus.status required: policy, health, controllers, logs or state" .Values.endpointStatus.status | quote }}
```

Comment:
would be nice to give an example with 2 or mor options set. for example: `"endpointStatus.status required: policy, health, controllers, logs and / or state. For 2 or more options use a comma: "policy, health""`

Reply:
It's a good example. Updated.